### PR TITLE
Use proper perl "signatures" in common files - commands

### PR DESCRIPTION
**Pull request by okurz**
Using a command like

```
sed -i "/^use strict;/ {N;s/use strict;/use Mojo::Base -strict, -signatures;/}" $(git ls-files)
for i in $(git grep -l 'use \(strict\|warnings\)'); do grep -q 'use Mojo::Base -strict' $i && sed -i '/use \(strict\|warnings\)/d' $i; done
tools/tidy --only-changed
```

and some manual fixes.

### CODE REVIEW

File: commands.pm [hunk 1]
```diff
@@ -20,8 +20,7 @@ BEGIN {
     $ENV{MOJO_REACTOR} = 'Mojo::Reactor::Poll';
 }
 
-# Automatically enables "strict", "warnings", "utf8" and Perl 5.10 features
-use Mojolicious::Lite;
+use Mojolicious::Lite -signatures;
```
Comment (perlpunk) [inline commands.pm:23]: why do we need to enable signatures twice?

Comment (okurz) [reply commands.pm:23]: I don't know. Do you think we should just remove this line?

Comment (okurz) [reply commands.pm:23]: If I remove the import then t/07-commands.t fails on unknown definition of "app". If I remove the `-signatures` flag then we get "The signatures feature is experimental"

Comment (Martchus) [reply commands.pm:23]: Yes, we're using `Mojolicious::Lite` here.

> If I remove the -signatures flag then we get "The signatures feature is experimental"

Maybe that module is special then. So I guess we can just keep the `-signatures` then.

**Review by Martchus — APPROVED** [review]

File: commands.pm (hunk 1 above)
Comment (perlpunk) [reply commands.pm:23]: hm, ok. I guess it doesn't hurt to have it twice

**Review by perlpunk — APPROVED** [review]

**Pull request merged**